test/bin/run-integration-tests.sh

```diff
@@ -48,6 +48,7 @@ PYTHONPATH="$( pwd )" \
     python test/bin/inject_api_key.py admin@user.com \
     "XZpXI40Uk85eozjQkU1zHJ6yZHpix+j0mo1TMeGZ4dPzIqVPVGPmyfeK"
 
+PYTHONPATH="$( pwd )" \
```

Comment:
@nagem One option is to handle the log db uri the same way as the scitran db uri. Not rely on `config` but rather a separate var that is passed in ala https://github.com/scitran/core/blob/master/test/integration_tests/python/conftest.py#L30

I'd prefer the 2 db uris are handled consistently.